content/10.diagnostics.md

```diff
@@ -47,9 +47,12 @@ To distinguish past infection from vaccination, serological tests detect antibod
 They are useful for collecting population-level information for epidemiological analysis, as they can be used to estimate the extent of the infection in a given area.
 Thus, serological tests may be useful to address population-level questions, such as the percent of cases that manifest as severe versus mild and for guiding public health and economic decisions regarding resource allocation and counter-disease measures.
 
-Molecular and serological tests therefore offer distinct, complementary perspectives on viral presence.
-Some of the same technologies are useful to both strategies.
-They have been employed to varying extents throughout the world since the start of the COVID-19 pandemic.
+Molecular and serological tests therefore offer distinct, complementary perspectives on COVID-19 infections.
+Some of the same technologies are useful to both strategies, and different technologies have been employed to varying extents throughout the world since the start of the COVID-19 pandemic.
+Two of the primary metrics used to evaluate these tests are sensitivity and selectivity.
+Sensitivity refers to a test's ability to correctly identify a true positive; for example, a test with 50% sensitivity might identify SARS-CoV-2 in only one of every two positive samples.
+On the other hand, selectivity refers to how accurately a test is able to identify a negative sample as negative.
+Here, this metric can be relevant both in terms of understanding the risk of false positives and in discussing whether a test is susceptible to identifying other coronaviruses.
```

Comment:
```suggestion
Two of the primary metrics used to evaluate these tests are sensitivity and specificity.
Sensitivity refers to a test's ability to correctly identify a true positive; for example, a test with 50% sensitivity would identify SARS-CoV-2 in only one of every two positive samples.
On the other hand, specificity refers to how well a test is able to identify a negative sample as negative.
This metric can be relevant both in terms of understanding the risk of false positives and in discussing whether a test is susceptible to identifying other coronaviruses.
```
@cbrueffer edited selectivity to specificity throughout, and I agree with that change.